app/models/setting.rb

```diff
@@ -208,7 +208,7 @@ def self.create!(opts)
   def self.regexp_expand_wildcard_string(string, options = {})
     prefix = options[:prefix] || '\A'
     suffix = options[:suffix] || '\Z'
-    prefix + Regexp.escape(string).gsub('\*', '.*') + suffix
+    prefix + Regexp.escape(string).gsub('\*', '.*').gsub('\?', '.?') + suffix
```

Comment:
I think this should be 
```suggestion
    prefix + Regexp.escape(string).gsub('\*', '.*').gsub('\?', '.') + suffix
```
because `.?` means 0 or 1 characters, while `.` is any single character - which iiuc is the idea for the `?` wildcard in the settings.